ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/batch/ConflictDetectionSpec.scala

```diff
@@ -262,13 +263,16 @@ class ConflictDetectionSpec extends AsyncWordSpec with Matchers with Inside with
   }
 
   private val aTransactionLogEntry = {
-    val builder = DamlLogEntry.newBuilder
+    val transaction = TransactionOuterClass.Transaction
```

Comment:
I would like a comment stating the transaction is invalid.

Reply:
Could we just set some random bytes instead? If it's invalid, do we need to build the object at all?

For example, would this work?

```scala
    logEntryBuilder.getTransactionEntryBuilder
      .setLedgerEffectiveTime(com.google.protobuf.Timestamp.getDefaultInstance)
      .setRawTransaction(ByteString.copyFromUtf8("invalid transaction"))
```

Reply:
Yup, it works, thanks.